app/src/main/java/com/concordium/wallet/ui/cis2/SendTokenActivity.kt

```diff
@@ -215,10 +222,14 @@ class SendTokenActivity : BaseActivity() {
         }
 
     private val getResultRecipient =
-        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { it ->
```

Comment:
As you're defining an explicit var, maybe name it something more descriptive than `it`?

Reply:
ok